packages/babel-helpers/src/helpers.js

```diff
@@ -413,6 +416,49 @@ helpers.interopRequireWildcard = template(`
   })
 `);
 
+helpers.specRequireInterop = template(`
+  (function (obj) {
+    if (obj && obj.__esModule) {
+      return obj;
+    } else {
+      var newObj = Object.create ? Object.create(null, {
+          default: {
+            value: obj,
+            writable: true,
+            enumerable: true
+          },
+          __esModule: {
+            value: true
+          }
+        }) : {
+          default: obj,
+          __esModule: true
+        };
+      if (typeof Symbol === "function" && Symbol.toStringTag) {
+        Object.defineProperty(newObj, Symbol.toStringTag, { value: "Module" })
+      }
+      return (Object.freeze || Object)(newObj);
+    }
+  })
+`);
+
+helpers.specImportCheck = template(`
+  (function (module, imports) {
+    if (!module.__esModule) throw new Error("Only ES modules can be checked");
```

Comment:
The `specInteropRequire` that is used in all imports wraps CJS modules in an ES module.